homeassistant/components/vesync/fan.py

```diff
@@ -22,7 +22,7 @@
 }
 
 SPEED_AUTO = "auto"
```

Comment:
Please remove this constant and its use in the `speed` property too.

Reply:
Do you mean to remove this block completely or just don't use a constant:

```
        if self.smartfan.mode == SPEED_AUTO:
            return SPEED_AUTO
```

Reply:
We should return `None` in that case. Use a different constant for the vesync smartfan value that doesn't start with `SPEED_`.